common/src/main/java/com/wynntils/models/ingredients/type/IdentificationModifier.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.ingredients.type;
+
+public enum IdentificationModifier {
+    INTEGER(""),
+    PERCENTAGE("%"),
+    FOUR_SECONDS("/4s"),
+    THREE_SECONDS("/3s"),
+    TIER(" tier");
+
+    private final String inGame;
+
+    IdentificationModifier(String inGame) {
+        this.inGame = inGame;
+    }
+
+    public String getInGame(String name) {
+        if (this != FOUR_SECONDS) return inGame;
+
+        if (name.equals("manaRegen")) return "/5s";
+        return "/3s";
```

Comment:
DIdn't you just say you removed this witchcraft in part 1? Am I completely losing it?

Reply:
You are looking at ingredients. I have not converted them yet, just moved the code from the old "GearProfile" things over here. (But in this split-up review it looks like added code, rather than moved)

Reply:
Oh, I have missed the fact this is ingredients... Got it!